src/main/java/unicash/logic/parser/GetCommandParser.java

```diff
@@ -30,5 +31,26 @@ public GetCommand parse(String args) throws ParseException {
         }
     }
 
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof GetCommandParser)) {
+            return false;
+        }
+
+        return other instanceof GetCommandParser;
```

Comment:
Same here, but just to check is there a reason we need to ensure quality for the same CommandParser? I don't think we have any requirement for this right